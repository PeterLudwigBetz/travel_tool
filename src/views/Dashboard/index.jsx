import React, { Component } from 'react';
import Base from '../Base';
import DashboardHeader from '../../components/DashboadHeader';

export default class Dashboard extends Base {
  render() {
    return (
      <DashboardHeader />
    );
  }
}
