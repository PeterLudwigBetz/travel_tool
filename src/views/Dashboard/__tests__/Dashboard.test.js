import React from 'react';
import { shallow } from 'enzyme';
import Dashboard from '..';

describe('<Dashboard />', () => {
  it('renders as expected', () => {
    const wrapper = shallow(<Dashboard />);
    expect(wrapper).toMatchSnapshot();
  });
})