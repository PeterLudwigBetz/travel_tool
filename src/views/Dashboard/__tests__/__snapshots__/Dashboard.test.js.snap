// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Dashboard /> renders as expected 1`] = `<DashboardHeader />`;
